pkgs/applications/misc/ranger/default.nix

```diff
@@ -1,9 +1,13 @@
 { stdenv, lib, fetchFromGitHub, python3Packages, file, less, highlight
-, imagePreviewSupport ? true, w3m ? null}:
+, imagePreviewSupport ? true, w3m ? null
+, pdfPreviewSupport ? true, poppler_utils ? null
```

Comment:
It'd be great if you could add support for `mutool` too, we already do this for text previews. I'll open an issue on this over at ranger/ranger.